pkg/server/options/authentication.go

```diff
@@ -43,7 +43,7 @@ const (
 	// A shard admin being member of the privileged system group.
 	// This will bypass most kcp authorization checks.
 	shardAdminUserName = "shard-admin"
-	// A kcp admin being member of system:kcp:clusterworkspace:admin and system:kcp:clusterworkspace:access.
+	// A kcp admin being member of system:kcp:workspace:admin and system:kcp:workspace:access.
```

Comment:
I found `system:kcp:workspace:access` but haven't found the admin one, is this comment up to date?

Reply:
IIRC those should be removed in favor of the new authorizers - let's open a follow-up issue and ask @s-urbaniak to clarify?